src/System.Device.Gpio/System/Device/ComponentInformation.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace System.Device
+{
+    /// <summary>
+    /// A class to provide informational data about system components and active drivers and bindings.
+    /// An instance of this class is typically obtained by calling QueryComponentInformation on any supported object.
+    /// The structure represents a tree of connected devices, e.g. a controller and its associated drivers.
```

Comment:
[Triage] Add note in all new APIs in System.Device.Gpio that API is experimental or for debugging purposes and that may change in the future